garage/experiment/local_tf_runner.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, sess=None, max_cpus=1):
         if max_cpus > 1:
             from garage.sampler import singleton_pool
             singleton_pool.initialize(max_cpus)
-        self.sess = sess or tf.Session()
+        self.sess = sess or tf.Session(config=tf.ConfigProto(use_per_session_threads=True))
```

Comment:
@naeioi  I think this is testable.

@gautams3 can you provide a minimal launcher which reproduces the problem?

Reply:
This needs a sampler and a LocalRunner for the sampler to call `start_worker`. One option is the [point_reach_embed launcher in embed2learn](https://github.com/ryanjulian/embed2learn-private/blob/388934023af752f5137791319f26164bebdae256/launchers/point_reach_embed.py)

Reply:
Could you grant me access to embed2learn? I cannot open the link above.

Reply:
done